packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.classnames.ts

```diff
@@ -2,42 +2,39 @@ import { cva, VariantProps } from 'class-variance-authority';
 import { ExcludeAllNull } from '@deriv/quill-design';
 
 export const JurisdictionCardClass = cva(
-    'items-center rounded-800 border-sm border-system-light-secondary-background cursor-pointer flex flex-col justify-center w-full lg:w-1/4 relative h-full transition-shadow duration-300 transform-gpu',
+    'items-center rounded-800 border-sm border-solid border-system-light-secondary-background cursor-pointer flex flex-col justify-center w-full lg:w-1/4 relative h-full transition-shadow duration-300 transform-gpu',
     {
         variants: {
             isAdded: {
                 true: 'cursor-not-allowed select-none',
             },
             isFlipped: {
-                true: 'rotate-180',
+                true: '[transform:rotateX(-180deg)]',
             },
             isSelected: {
-                true: 'border-sm border-solid border-system-light-primary-background',
+                true: 'border-[#85acb0] shadow-[0_24px_48px_0_rgba(14,_14,_14,_0.18)]',
```

Comment:
Looks good but can we incorporate the colors from `@deriv/quill-design` or from our `tailwind.config`. If it doesn't exist we can always add them to the file.

Reply:
Updated.